### ISSUE TITLE
FAP General Tab Accessibility Issues

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1659]
The following are issues on the General tab on the FAP page that are preventing WCAG compliance:
- The "Review visibility" label is orphaned from input
- When the "Custom Grade Guide" checkbox is checked, the textbox that appears doesn't get focus, and it comes before the checkbox in focus order. This means keyboard and screen reader users may miss it.
- The fields should be more descriptive e.g. Code -> FAP Code, Description -> FAP Description, Active checkbox -> Is FAP active
